scripts/ci/verify-build.js

```diff
@@ -45,9 +45,11 @@ async function start() {
  *
  * @param {VerificationOptions} options
  */
-async function verifyBuild( { language, packageManager, customPluginName, useOnlyNewInstallationMethods } ) {
+async function verifyBuild( { language, packageManager, customPluginName, installationMethod } ) {
 	let testSetupInfoMessage = `Testing build for language: [${ language }] and package manager: [${ packageManager }]`;
 
+	const IsSupportingOnlyCurrentInstallationMethod = installationMethod === 'current';
```

Comment:
Why `PascalCase` instead of our usual `camelCase`? I also think that flipping the flag would make more sense.

```suggestion
	const supportsLegacyMethods = installationMethod !== 'current';
```

Same for changes in `packages/ckeditor5-package-generator/lib/utils/copy-files.js`

Reply:
Addressed in 4a0a6f5.